app/views/nooks/index.html.erb

```diff
@@ -1,15 +1,41 @@
 <%# nooks#index - app/views/nooks/index.html.erb %>
 
 <% @nooks.each do |nook| %>
-<p><%= link_to nook.name, nook_path(nook) %></p>
+  <p><%= link_to nook.name, nook_path(nook) %></p>
 <% end %>
 
 <%= link_to "nooks#new", new_nook_path %>
-<%# <%= link_to "profiles#show", profiles_show_path %> %>
```

Comment:
we won't need these in the future anyhow, since they'll be rendered in the directory partial.